medusa/tv/series.py

```diff
@@ -559,22 +559,19 @@ def last_update_indexer(self):
     @property
     def prev_aired(self):
         """Return last aired episode ordinal."""
-        today = datetime.date.today().toordinal()
-        if not self._prev_aired or self._prev_aired > today:
-            if self._last_update_indexer >= today:
-                self._prev_aired = self.prev_episode()
+        self._prev_aired = self.prev_episode()
         return self._prev_aired
 
     @property
     def next_aired(self):
         """Return next aired episode ordinal."""
         today = datetime.date.today().toordinal()
         if not self._next_aired or self._next_aired < today:
-            if self._last_update_indexer >= today:
-                self._next_aired = self.next_episode()
+            self._next_aired = self.next_episode()
```

Comment:
Shouldn't next_episode() be cached as well? E.g. when a season is over and the next is coming in a year or so. It would constantly trigger. Maybe with a a longer TTL like 12h?

Reply:
I did use next_airdate in stead.